conn/physic/units.go

```diff
@@ -154,6 +154,47 @@ func (e ElectricResistance) String() string {
 	return nanoAsString(int64(e)) + "Ω"
 }
 
+// Set sets the ElectricResistance to the value represented by s. Units are to
+// be provided in "Ohm", "Ohms" or "Ω" with an optional SI prefix: "n", "p", "u"
+// , "µ", "m", "k", "M", "G" or "T".
+func (f *ElectricResistance) Set(s string) error {
+	v, n, err := valueOfUnitString(s, nano)
+	if err != nil {
+		if e, ok := err.(*parseError); ok {
+			switch e.err {
+			case errOverflowsInt64:
+				return errors.New("maximum value is " + maxElectricResistance.String())
+			case errUnderflowsInt64:
+				return errors.New("minimum value is " + minElectricResistance.String())
+			case errNotANumber:
+				if found, _ := containsUnitString(s, "Ohm", "Ohms", "Ω"); found != "" {
+					return errors.New("does not contain number")
+				}
+				return errors.New("does not contain number or unit \"Ohm\"")
+			}
+		}
+		return err
+	}
+
+	if s[n:] == "Ω" {
```

Comment:
The reason I did `rest := s[n:]` is that I don't think the Go compiler will be smart enough to optimize the two construction, especially with bound checks.